tests/dummy/app/pods/application/template.hbs

```diff
@@ -1,8 +1,4 @@
-<div class="demo-notification-container">
-  {{#each notifications as |notification|}}
-    {{notification-message notification=notification}}
-  {{/each}}
-</div>
+{{notification-container zindex=9999}}
```

Comment:
Doesn't `zindex` need a `-`?

Reply:
It does in plain css

Reply:
This isn't setting the style or CSS property, though, it's an argument to the Ember notification widget. The widget now sets its z-index explicitly with a `style=`, and the way to override that is with a component property named `zindex`.

http://stonecircle.github.io/ember-cli-notifications/

https://github.com/stonecircle/ember-cli-notifications/blob/086951c588867c165f8776abb2fde769d470b31c/addon/components/notification-container.js#L14